src/NeonView.js

```diff
@@ -63,25 +63,27 @@ class NeonView {
         this.updateForCurrentPage();
       }
     }); */
-    this.core.initDb().then(() => { this.updateForCurrentPage(); });
+    this.core.initDb().then(() => { this.updateForCurrentPage(true); });
   }
 
   /**
    * Get the current page from the loaded view and then display the
    * most up to date SVG.
+   * @param {boolean} delay. whether or not to delay loading the page by 500ms. defaults to false
```

Comment:
Correct should be `@param {boolean} [delay=false] - whether or not...`

Reply:
Also is this 500 ms hard coded? If it is put in a variable somewhere, reference the variable instead of the value (which could change).

Reply:
it's not hard coded it refers to this.loadDelay in divaview.js